app/models/game.rb

```diff
@@ -11,14 +11,15 @@ class Game < ApplicationRecord
   # --------------------
   scope :for_location, -> (location_id) { where("location_id = ?", location_id) }
   scope :for_sport, -> (sport_id) { where("sport_id = ?", sport_id) }
-  scope :sort_time, -> { order('time') }
+  scope :not_for_sport, -> (sport_id) { where("sport_id != ?", sport_id) }
+  scope :sort_time, -> { order('time ASC') }
   scope :search, -> (term) { where("description LIKE ?", "%#{term}%")}
+  scope :sort_location, -> { order('location_id') }
```

Comment:
May need to make sort_location like alphabetical cause would the ID be related to the name or actual coordinates?